sklearn/decomposition/sparse_pca.py

```diff
@@ -239,6 +284,20 @@ class MiniBatchSparsePCA(SparsePCA):
         If None, the random number generator is the RandomState instance used
         by `np.random`.
 
+    normalize_components : boolean
```

Comment:
I would copy paste the previous docstring of sparsepca. You need to update the couple of the above nitpicks.